example/rnn/bucketing/cudnn_gru_bucketing.py

```diff
@@ -0,0 +1,235 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
don't copy a whole file. rename cudnn_lstm_bucketing.py to cudnn_rnn_bucketing.py and then add a switch for the mode instead.

Reply:
fixed, thanks!